magic_numbers.py

```diff
@@ -15,5 +16,31 @@
 # Mask erode/dilate kernel
 ERODE_DILATE_KERNEL = np.ones((3, 3), dtype=np.float32)
 
+MIN_ASPECT_RATIO = 0.9
+
 # Factor by which areas of contours are multiplied to get an estimate for allowed distance squared between them
-METRIC_SCALE_FACTOR = 27.5
+METRIC_SCALE_FACTOR = 26
+
+# The following were careful measurements of the frame area with the camera
+# aimed at a flat wall, and the distance of the camera from the wall. All are in
+# millimetres.
+FOV_WIDTH = 1793
+FOV_HEIGHT = 2303
```

Comment:
we should double check these, it seems weird that there is a higher vertical fov that horizontal